### PR TITLE
Feature/951

### PR DESCRIPTION
**Pull request by xak**
#951

### CODE REVIEW

**Comment by xak** [comment]
This is ready for review at branch `feature/951` at http://localhost:3030/tree

**Note:** I did not include an example of the selected state but it is available. Just add `aria-selected="true"` or class ```.is-selected` to the approriate `__row` element. The coding to implement this for demo-able purposes alone is substantial and since it is not present in the existing component I skipped it.

**Comment by LeoT7508** [comment]
@xak 

The only issue I see here is the text doesn't seem to be centered vertically in the row. Theres less padding on above than below the text. 

screen shot 2017-10-09 at 2 31 47 pm

**Comment by xak** [comment]
@LeoT7508 Fixed. Pull the changes and check again.

File: scss/components/tree.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        //pad to account for control
+        padding-left: $_row_padding;
+        &--control {
+            flex: auto;
+            width: 25%;
+            //sets 8 sublevels of indentation based on __group level
+            @at-root {
+                @each $i in 1,2,3,4,5,6,7,8 {
+                    .#{$block}__group--sublevel-#{$i} & {
+                        //padding-left: ($tn-tree-cell-spacing * ($i + 1));
```
Comment (saadmhybris) [inline scss/components/tree.scss]: we can get rid of this commented line if no longer needed

**Pull request merged**